lib/pansophy_authenticator/configuration/base.rb

```diff
@@ -3,32 +3,26 @@
 
 module PansophyAuthenticator
   module Configuration
-    class Instance
-      include Anima.new :local, :bucket_name, :file_path, :application, :cache_store
+    class Base
+      include Anima.new :bucket_name, :file_path, :application, :cache_store
       include Memoizable
 
       def local?
-        @local
+        is_a?(Local)
```

Comment:
Expecting a parent class to know about its child classes is an anti-pattern imho